--- config_machines.xml 2015-11-17 17:23:03.000000000 +0000
+++ config_machines.xml.old     2015-11-17 17:13:42.000000000 +0000
@@ -569,23 +569,4 @@
         <MAX_TASKS_PER_NODE>32</MAX_TASKS_PER_NODE>
 </machine>
 
-<machine MACH="legion">
-         <DESC>UCL Legion, os is Linux, batch system is SGE, user-defined</DESC>
-         <EXEROOT>/home/$CCSMUSER/Scratch/cesm/cases/$CASE</EXEROOT>
-         <OBJROOT>$EXEROOT<OBJROOT>
-         <LIBROOT>$EXEROOT/lib<LIBROOT>
-         <INCROOT>$EXEROOT/lib/include<INCROOT>
-         <DIN_LOC_ROOT_CSMDATA>/home/$CCSMUSER/Scratch/cesm/inputdata<DIN_LOC_ROOT_CSMDATA>
-         <DIN_LOC_ROOT_CLMQIAN>/home/$CCSMUSER/Scratch/cesm/inputdata/atm/datm7/atm_forcing.datm7.Qian.T62.c080727<DIN_LOC_ROOT_CLMQIAN>
-         <DOUT_S_ROOT>/home/$CCSMUSER/Scratch/cesm/archive/$CASE<DOUT_S_ROOT>
-         <DOUT_L_HTAR>FALSE<DOUT_L_HTAR>
-         <DOUT_L_MSROOT>UNSET<DOUT_L_MSROOT>
-         <OS>Linux<OS>
-         <BATCHQUERY>qstat<BATCHQUERY>
-         <BATCHSUBMIT>qsub<BATCHSUBMIT>
-         <GMAKE_J>1<GMAKE_J>
-         <MAX_TASKS_PER_NODE>4<MAX_TASKS_PER_NODE>
-         <MPISERIAL_SUPPORT>FALSE<MPISERIAL_SUPPORT>
-</machine>
-
 </config_machines>
