--- install_elpa.sh.orig	2020-01-07 14:52:49.000000000 +0000
+++ install_elpa.sh	2020-01-07 14:48:01.000000000 +0000
@@ -75,9 +75,9 @@
             has_GPU=$([ "$ENABLE_CUDA" == "__TRUE__" ] && echo "yes" || echo "no")
             FMA_flag=`grep '\bfma\b' /proc/cpuinfo 1>/dev/null && echo '-mfma' || echo '-mno-fma'`
             SSE4_flag=`grep '\bsse4_1\b' /proc/cpuinfo 1>/dev/null && echo '-msse4' || echo '-mno-sse4'`
-            grep '\bavx512dq\b' /proc/cpuinfo 1>/dev/null && AVX512_flags+=" -mavx512dq"
+#            grep '\bavx512dq\b' /proc/cpuinfo 1>/dev/null && AVX512_flags+=" -mavx512dq"
             grep '\bavx512cd\b' /proc/cpuinfo 1>/dev/null && AVX512_flags+=" -mavx512cd"
-            grep '\bavx512bw\b' /proc/cpuinfo 1>/dev/null && AVX512_flags+=" -mavx512bw"
+#            grep '\bavx512bw\b' /proc/cpuinfo 1>/dev/null && AVX512_flags+=" -mavx512bw"
             grep '\bavx512v1\b' /proc/cpuinfo 1>/dev/null && AVX512_flags+=" -mavx512v1"
             # non-threaded version
             mkdir -p obj_no_thread; cd obj_no_thread
@@ -88,7 +88,6 @@
                           --enable-static=yes \
                           --enable-avx=${has_AVX} \
                           --enable-avx2=${has_AVX2} \
-                          --enable-avx512=${has_AVX512} \
                           --enable-gpu=${has_GPU} \
                           --with-cuda-path=${CUDA_PATH} \
                           FC=${MPIFC} \
@@ -113,7 +112,6 @@
                               --enable-static=yes \
                               --enable-avx=${has_AVX} \
                               --enable-avx2=${has_AVX2} \
-                              --enable-avx512=${has_AVX512} \
                               --enable-gpu=${has_GPU} \
                               --with-cuda-path=${CUDA_PATH} \
                               FC=${MPIFC} \
