--- matlab/SearchProjectionOnPial.m.old 2015-09-21 15:01:23.736328806 +0100
+++ matlab/SearchProjectionOnPial.m     2015-09-21 15:02:12.666328780 +0100
@@ -4,7 +4,8 @@
 % limit redundancies in the resulting path file.
 
 verticeslist=[];
-for t=1:step:size(perim,2)
+si=max(size(perim));
+for t=1:step:si
     [nearestIndexMT,nearestValuesMT]=mesh_vertex_nearest(mesh_total.vertices,mesh_outer.vertices(perim(t),:));
     verticeslist= [verticeslist nearestIndexMT];
 end
