--- makeQt	2014-02-06 15:31:37.000000000 +0000
+++ makeQt.new	2015-05-01 16:32:40.926838372 +0100
@@ -29,7 +29,7 @@
 #     Build script for qt-everywhere-opensource-src
 #
 #------------------------------------------------------------------------------
-qtVERSION=4.7.3
+qtVERSION=4.8.6
 qtTYPE=qt-everywhere-opensource-src
 #------------------------------------------------------------------------------
 # run from third-party directory only
@@ -121,11 +121,12 @@
 
     # compile as opensource, accepting LGPL conditions
     echo "yes" | \
-    $sourceDIR/configure \
+    $sourceDIR/configure --no-pch \
         --prefix=$prefixDIR \
         -opensource \
         -nomake demos \
         -nomake examples \
+        -lirc -limf  \
     && time make -j $WM_NCOMPPROCS \
     && make install \
     && echo "Built: Qt-$qtVERSION" \
