--- g09/linda8.2/opteron-linux/bin/LindaLauncher.old    2015-09-24 10:19:15.000000000 +0100
+++ g09/linda8.2/opteron-linux/bin/LindaLauncher        2015-09-24 10:21:15.000000000 +0100
@@ -1,6 +1,6 @@

-LINDA_PATH=/dev/shm/g09/linda8.2/opteron-linux/
-export LINDA_PATH
+#LINDA_PATH=/dev/shm/g09/linda8.2/opteron-linux/
+#export LINDA_PATH

 # Some systems expand quoted null arg lists into the null string

