--- makeQt	2014-02-06 15:31:37.000000000 +0000
+++ makeQt.new	2015-05-01 14:28:13.785241415 +0100
@@ -29,7 +29,7 @@
 #     Build script for qt-everywhere-opensource-src
 #
 #------------------------------------------------------------------------------
-qtVERSION=4.7.3
+qtVERSION=4.8.6
 qtTYPE=qt-everywhere-opensource-src
 #------------------------------------------------------------------------------
 # run from third-party directory only
